kube_hunter/modules/hunting/kubelet.py

```diff
@@ -244,7 +244,7 @@ class DebugHandlers:
         """ all methods will return the handler name if successful """
 
         def __init__(self, path, pod, session=None):
-            self.path = path
+            self.path = path + "/" if not path.endswith("/") else ""
```

Comment:
Hmm...I think this is going to leave path as an empty string if it does end with /
```
>>> p = "hello/"
>>> p = p + "/" if not p.endswith("/") else ""
>>> p
''
```

Some extra brackets will fix it
```
>>> p = "hello/"
>>> p = p + ("/" if not p.endswith("/") else "")
>>> p
'hello/'
```

Reply:
Youre right!